new/collect/check-git-status.ts

```diff
@@ -25,25 +25,29 @@ export enum exitCodes {
  * checks if user is on the right branch and has everything committed
  */
 export const checkGitStatus: Module = async (env: Env) => {
-  // exit if there are still files to commit (unless in verify mode)
-  if (!isVerify(env) && (env.uncommittedFiles?.length ?? 0) > 0) {
+  log(`running on branch ${bold(env.currentBranch)}`);
+
+  // Notify the user if there are still files to commit
+  if (env.uncommittedFiles && env.uncommittedFiles.length > 0) {
```

Comment:
I like that idea!
Now it also shows the log messages in case of errors, and only exists when `!isVerify(env)`.

btw, `process.exit()` only ? Should we also (instead?) throw exception like other modules?

Reply:
I guess I took that over from the older implementations. So we could align and end with an error.

Reply:
I'll create a follow up topic to align this